src/components/CookieConsent.astro

```diff
@@ -0,0 +1,157 @@
+<div class="consent-banner">
+  <div class="consent-info">
+    <p>Can we store cookies?</p>
+    <p>
+      By clicking “Accept”, you agree to the storing of cookies on your device
+      to analyse site usage.
+    </p>
+  </div>
+  <div class="consent-controls">
+    <a href="/privacy">Privacy Policy</a>
```

Comment:
Can we add the privacy link in the text? somethink like: "Please consult our `privacy policy` to learn more about this."